src/client/article.ts

```diff
@@ -436,6 +438,19 @@ function richLinks(): void {
 		});
 }
 
+function hydrateClickToView(): void {
+	document.querySelectorAll('.click-to-view-container').forEach((container) =>
```

Comment:
Could we prefix this class with `js-` just to make clear its purpose is to be used by JavaScript?

```suggestion
	document.querySelectorAll('.js-click-to-view-container').forEach((container) =>
```